Makefile

```diff
@@ -347,7 +347,7 @@ ifneq ($(AVR), 0)
 	@$(MD5SUM) test.hex
 endif
 ifneq ($(XTENSA), 0)
-	$(TINYGO) build -size short -o test.bin -target=esp32               examples/serial
+	$(TINYGO) build -size short -o test.bin -target=esp32-wroom-32      examples/blinky1
```

Comment:
esp-wroom-32 might be a better choice.

Reply:
It appears that this is the old name.
I can't find esp-wroom-32 on their product page: https://www.espressif.com/en/products/modules/esp32
Wikipedia claims (without citation) that it is the old name:

> First publicly available ESP32 module board created by Espressif.[11] FCC Part 15.247 tested (FCC ID: 2AC7Z-ESPWROOM32).[12] Based on ESP32-D0WDQ6 chip. Originally named "ESP-WROOM-32".

Reply:
You're right.